src/coreComponents/python/modules/pygeosx_tools_package/pygeosx_tools/file_io.py

```diff
@@ -0,0 +1,53 @@
+
+import os
+import numpy as np
+
+
+def save_tables(grid, properties, table_root='./tables', axes=['x', 'y', 'z', 't']):
+    """
+    Saves a set of tables in GEOSX format
+    Notes: The shape of these arrays should match the length of each axis in the specified order
+           The output directory will be created if it does not exist yet
+
+    Args:
+        grid (list): A list of numpy ndarrays defining the table axes
```

Comment:
`grid` being an iterable, don't you think you could give a plural name? (maybe `grids`, but I am not sure).

Reply:
Agreed, I've changed the following to be more clear:

-  "grid" to "axes"
- "axes" to "axes_names"